### DISCUSSION TITLE
Validated example of in-network-rates file with external provider-reference files

### DISCUSSION THREAD

**Discussion by olderthandirt5** [CMSgov/price-transparency-guide#480] [Show and tell]
Since there are limited real world samples available, here is a very small example.

The attached sample shows how we are doing in-network-rates with external provider-reference files. It has passed validation.
[Converted to a .txt extension to allow for upload.]

> `in-network-rates_SAMPLE.json v1.0.0 -t in-network-rates
> 
> Input JSON is valid.
> `
To test it in your own environment, just download the file and change the extension to ".json".
[in-network-rates_SAMPLE.txt](https://github.com/CMSgov/price-transparency-guide/files/8682958/in-network-rates_SAMPLE.txt)

Hope that helps.

**Comment by shaselton-usds** [comment]
@olderthandirt5 Thanks for providing your example -- the more of these we have, the better idea we'll all have on what to expect.

**Comment by annesig** [comment]
This is just what I was looking for earlier this week!  Thanks so much for posting this!

**Comment by shaselton-usds** [comment]
@olderthandirt5 Maybe this is just a sample file, but I would point out that the `provider_references` inside of the `negotiated_rates` array doesn't reference a valid `provider_group_id`. You have "10" and "20" when you only have "1" and "2" defined in your `provider_group_id` attributes.

**Comment by olderthandirt5** [comment]
Yes, this was just an example.
